examples/custom-tags/templates/index.html

```diff
@@ -0,0 +1,4 @@
+<script type="text/javascript"></script>
```

Comment:
Without it, the pipe script would be inserted after the switcher and won't be available to it's content. @vigneshshanmugam mb you know the nice way to do it?

Reply:
Yeah pipe is inserted before `fragment` tag or one of the tags in handleTags list. Just pass them in the example :)

Reply:
You mean `pipeTags`?